tests/profiling/_test_multiprocessing.py

```diff
@@ -8,6 +9,7 @@ def f():
 
 
 if __name__ == "__main__":
+    print(os.getpid())
```

Comment:
Just checking this was not accidentally forgotten here

Reply:
No, I changed the way of getting the process pid.
Windows uses a different mechanism and the pid returned by subprocess is not the one we want for the test.